Automattic/wp-calypso
# Email verification banner - add basic unit tests.

Related to # https://github.com/Automattic/wp-calypso/pull/94683

## Proposed Changes

* Expands on the above PR to add basic unit tests for the email verification banner.
    * It should appear when email is not verified.
    * It should not appear when email is verified and there is no pending change.
    * It should appear if there is a pending email change.
    * Clicking the cta should open the dialog modal. 

## Why are these changes being made?


* Unit tests are good. A recent change above removed a component that had unit tests, and replaced it with a new component. Lets add unit tests for the new component too...

## Testing Instructions



* Check that the unit tests are running in automated flows below and passing.
* Review the tests locally and ensure they make sense. you can run them with `yarn test-client -- client/me/email-verification-banner/test/`

## Pre-merge Checklist
